src/main/java/seedu/cafectrl/command/ExitCommand.java

```diff
@@ -0,0 +1,13 @@
+package seedu.cafectrl.command;
+
+import seedu.cafectrl.data.Menu;
+import seedu.cafectrl.ui.Ui;
+
+public class ExitCommand extends Command{
+    public static final String COMMAND_WORD = "bye";
+    @Override
```

Comment:
Maybe you can have one empty line here to make a distinction between attributes and method